src/index.mako.html

```diff
@@ -717,9 +721,9 @@
         module.constant('gaGlobalOptions', {
           //dev3d to be removed once 3d goes live
 % if staging=='prod' :
-          dev3d: false,
+          dev3d: true,
```

Comment:
Please revert this for this branch so we can merge now. Final activation will be on 3dalpha branch.